src/assets/src/components/common.tsx

```diff
@@ -247,21 +255,29 @@ export const StatelessInputGroupForm: React.FC<StatelessValidatedInputFormProps>
                 <Form.Control
                     id={props.id}
                     as='input'
+                    ref={inputRef}
                     className='form-control-remaining'
                     value={props.value}
                     placeholder={props.placeholder}
                     onChange={(e: any) => handleChange(e.currentTarget.value)}
                     disabled={props.disabled}
                     isInvalid={props.validationResult?.isInvalid}
                 />
-            {buttonBlock}
+                {buttonBlock}
             </InputGroup>
             {feedback}
         </Form>
     );
 }
 
 export const StatelessTextAreaForm: React.FC<StatelessValidatedInputFormProps> = (props) => {
+    const inputRef = createRef<FormControl<"textarea"> & HTMLTextAreaElement>();
+    useEffect(() => {
+        if (props.initFocus) {
+            inputRef.current!.focus();
+        }
+    }, []);
+
```

Comment:
What do you think about making a hook that creates the ref, sets up the effect, and returns the ref?

Reply:
Let me see what it looks like.

Reply:
@jlost, how about this? https://github.com/tl-its-umich-edu/remote-office-hours-queue/pull/194/commits/79d9792c37e66895fdc5fcf1329bebabc5a1e32e

I'm still getting comfortable with generics and these custom hooks. It seemed to work well to rely on passing a type to `<>` as part of the function calls, but I'm not entirely clear on whether that's a standard practice. It does seem to be how `createRef` works.

Reply:
I made a couple other edits, so I'm going to re-request your review.